tests/actionsFactory.spec.ts

```diff
@@ -52,16 +50,16 @@ describe("actionsFactory test", () => {
 
         it("fails without name", () => {
             const actionJson = {
-                [ActionJsonKey.ActionType]: ActionType.Command
+                actionType: "COMMAND"
             };
-            expect(() => ActionsFactory.createAction(actionJson)).to.throw(`${ActionJsonKey.CommandName} is missing for actionType=${ActionType.Command}`);
+            expect(() => ActionsFactory.createAction(actionJson)).to.throw(`commandName is missing for actionType = COMMAND`);
```

Comment:
generally for these kinds of assertions:
- Consider using some constants to make future refactoring easier.
- Consider using less precise error message matching using a regular expression to reduce maintenance overhead.
  - https://www.chaijs.com/api/bdd/#method_throw